website_sale_product_attribute_filter_collapse/views/templates.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Tecnativa - Pilar Vargas
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template
+        id="products_attributes_collapsible"
+        inherit_id="website_sale.products_attributes"
+        name="Filter by collapsible attributes"
+        active="False"
+        customize_show="True"
+        priority="999"
+    >
+        <!-- Add a new title zone -->
+        <xpath expr="//ul[hasclass('flex-column')]/t/li/div" position="before">
+            <a
+                data-toggle="collapse"
+                class="collapsed"
+                t-attf-href="#collapse_v_{{a.id}}"
+            >
+                <i
+                    class="fa fa-angle-right"
+                    id="icon_attribute_collapsed"
+                    aria-hidden="true"
+                />
+                <i
+                    class="fa fa-angle-down"
+                    id="icon_attribute_expanded"
+                    aria-hidden="true"
+                />
+            </a>
+        </xpath>
+        <!-- -->
+        <xpath
+            expr="//ul[hasclass('flex-column')]/t/li/div/strong"
+            position="attributes"
+        >
+            <attribute name="class">o_shop_collapse_attribute_filter</attribute>
+        </xpath>
+        <!-- Move attribute name to the title zone -->
```

Comment:
What title zone?

Reply:
To the title zone added above, I'll make it clearer in the comment :)